black-mulberry-rest/src/main/java/com/black/mulberry/rest/endoint/ProductEndpoint.java

```diff
@@ -0,0 +1,51 @@
+package com.black.mulberry.rest.endoint;
+
+import com.black.mulberry.core.entity.Product;
+import com.black.mulberry.core.exception.ProductNotExistException;
+import com.black.mulberry.core.mapper.ProductMapper;
+import com.black.mulberry.core.repository.ProductRepository;
+import com.black.mulberry.core.service.ProductService;
+import com.black.mulberry.data.transfer.request.ProductRequest;
+import com.black.mulberry.data.transfer.response.ProductResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/products")
+public class ProductEndpoint {
+    private final ProductRepository productRepository;
+    private final ProductService productService;
+    private final ProductMapper productMapper;
```

Comment:
why you authowired product mapper and productRepository?
1. its unused here.
2. lets make mapping/repo actions in service layer.